### PR TITLE
ESsdm021 texts

### PR DESCRIPTION
**Pull request by eu-genia**
(checking with SD whether/where separate work IDs for prayers must be created)

### CODE REVIEW

File: new/LIT5922Adam.xml [hunk 1]
```diff
@@ -0,0 +1,57 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT5922Adam" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">ስምዑ፡ ወልብው፡ ኦደቂቃ፡ አዳም፡</title>
```
Comment (thea-m) [inline new/LIT5922Adam.xml:8]: is it ደቂቃ in the manuscript?
ʾAddām?

File: new/LIT5923TenCommand.xml [hunk 2]
```diff
@@ -0,0 +1,56 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT5923TenCommand" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">ነገረ፡ በእንተ፡ አሠርቱ፡ ቃላት፡</title>
```
Comment (thea-m) [inline new/LIT5923TenCommand.xml:8]: it should be nagar, is it nagara in the manuscript?

Comment (eu-genia) [reply new/LIT5923TenCommand.xml:8]: This is the way Sophia catalogued in SDM-021

Comment (thea-m) [reply new/LIT5923TenCommand.xml:8]: I just checked, the manuscript has nagar, then we can correct it here

Comment (eu-genia) [reply new/LIT5923TenCommand.xml:8]: Checked in MS, it is nagar sure enough, thank you

**Comment by eu-genia** [comment]
Following the communication from @SophiaD-M - more new work records created

File: new/LIT5927PrayerBE.xml [hunk 3]
```diff
@@ -0,0 +1,67 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT5927PrayerBE" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">ይትባረክ፡ እግዚአብሔር፡ አምላከ፡ እስራኤል፡ ወዝንቱ፡ ውእቱ፡ ስማየ፡ ዘነገርክዎ፡</title>
+                <title xml:lang="gez" corresp="#t1" type="normalized">Yǝtbārak ʾǝgziʾabǝḥer ʾamlāka ʾǝsrāʾel wazǝntu wǝʾtu sǝmāya zanagarkwo...</title>
```
Comment (thea-m) [inline new/LIT5927PrayerBE.xml:9]: wǝʾǝtu
za-nagarkǝwo
ስምየ and sǝmǝya (I checked the manuscript)

File: new/LIT5927PrayerBE.xml [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+            </langUsage>
+        </profileDesc>
+        <revisionDesc>
+            <change who="ES" when="2020-05-20">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <div type="edition" xml:lang="gez">
+                <div type="textpart" subtype="incipit"><ab>ይትባረክ፡ እግዚአብሔር፡ አምላከ፡ እስራኤል፡ ወዝንቱ፡ ውእቱ፡ ስማየ፡ ዘነገርክዎ፡</ab>
```
Comment (thea-m) [inline new/LIT5927PrayerBE.xml:54]: see above

File: new/LIT5928IncantationBE.xml [hunk 5]
```diff
@@ -0,0 +1,67 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT5928IncantationBE" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">ይትባረክ፡ እግዚአብሔር፡ አምላከ፡ እስራኤል፡ ኤል፡ ኤል፡ ኤል፡ … ወእምብድብድ፡ ጽውዓ፡ ጽዋዕኩ፡</title>
```
Comment (thea-m) [inline new/LIT5928IncantationBE.xml:8]: እስራኤል፡ ኤል፡ ኪር፡ ኪር፡...in the manuscript
ጸዋዕኩ in the manuscript

File: new/LIT5930PrayerBE.xml [hunk 6]
```diff
@@ -0,0 +1,67 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT5930PrayerBE" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">ወይእዜኒ፡ እግዚኦ፡ እግዚአብሔር፡ አሜን፡ ቡሩክ፡</title>
+                <title xml:lang="gez" corresp="#t1" type="normalized">Wayʾzeni ʾǝgziʾo ʾǝgziʾabǝḥer ʾamen buruk</title>
```
Comment (thea-m) [inline new/LIT5930PrayerBE.xml:9]: yǝʾǝze

File: new/LIT5932PrayerBE.xml [hunk 7]
```diff
@@ -0,0 +1,67 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT5932PrayerBE" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">ተዘከር፡ እግዚአ፡ ኪዳነ፡ አብርሃም፡</title>
```
Comment (thea-m) [inline new/LIT5932PrayerBE.xml:8]: ተዘከር፡ እግዚኦ፡ ኪዳነ፡ አግብርቲከ፡ ኪዳነ፡ አብርሃም፡ in the manuscript

File: new/LIT5932PrayerBE.xml [hunk 8]
```diff
@@ -0,0 +1,67 @@
...
+                <p/>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p>A prayer used by the <orgName ref="ETH1274Betaes"/></p>
```
Comment (thea-m) [inline new/LIT5932PrayerBE.xml:34]: I think the use of orgName  hasn't been discussed yet. I would use persName here

File: new/LIT5925HymnBE.xml [hunk 9]
```diff
@@ -0,0 +1,67 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT5925HymnBE" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">ይትባረክ፡ እግዚአብሔር፡ አምላከ፡ እስራኤል፡ ወእዜኑ፡ ጽድቀከ፡ በአፉየ፡ ለትውልደ፡ ትውልድ፡ እስከ፡ ለዓለም፡</title>
+                <title xml:lang="gez" corresp="#t1" type="normalized">Yǝtbārak ʾǝgziʾabḥer ʾamlāka ʾǝsrāʾel waʾzenu ṣǝdǝqaka baʾafuya latǝwǝlda tǝwlǝd ʾǝska laʿālam...</title>
```
[the remark this answers is not in the document]
Comment (DariaElagina) [reply new/LIT5925HymnBE.xml:9]: Good question… I would be rather for ʾǝzenu

File: new/LIT5932PrayerBE.xml (hunk 8 above)
Comment (eu-genia) [reply new/LIT5932PrayerBE.xml:34]: I copied it from one of the records you edited or created

**Pull request merged**